packages/zoe/tools/setup-zoe.js

```diff
@@ -61,12 +61,31 @@ export const setUpZoeForTest = async ({
   );
 
   /**
-   * @param {string} path
+   * @param {object} pathOrExports
    * @returns {Promise<Installation>}
    */
-  const bundleAndInstall = async path => {
-    const bundle = await bundleSource(path);
-    const id = `b1-${path}`;
+  const bundleAndInstall = async pathOrExports => {
+    await null;
+    let id;
+    let bundle;
+    if (typeof pathOrExports === 'string') {
+      const path = pathOrExports;
+      bundle = await bundleSource(path);
+      id = `b1-${path}`;
+    }
+    {
+      assert.equal(
+        Object.prototype.toString.call(pathOrExports),
+        '[object Module]',
+      );
+      // Copy all the properties so this object can be hardened.
+      const exports = { ...pathOrExports };
+      bundle = harden({
+        moduleFormat: 'test',
+        [Symbol.for('exports')]: exports,
+      });
```

Comment:
Why not use [`bundleTestExports`](https://github.com/endojs/endo/pull/2735/files#diff-b93764e4d9804500e1cd218013ddf54177d51099601d766a974c5a11151c13fdR182-R190) ?